### PR TITLE
Addition of NewsLetter.

### PR DESCRIPTION
**Pull request by Chandan-Kumar-Mahato**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [ ] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [ ] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

**Comment by Chandan-Kumar-Mahato** [comment]
I have added all the newsLetter to the Ui/Ux section so that other can find it easily .

File: UI-UX Design/README.md [hunk 1]
```diff
@@ -289,8 +289,60 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
...
       </tr>
-      <tr>
-      <td></td>
-      <td></td>
-      </tr>
+      <!-- first one -->
+      <tr>
+      <td><a href="https://newsletter.uxdesign.cc/">UX Collective Newsletter</a></td>
+      <td>UX Collective is the premier Medium publication for UX content with a high editorial standard
+      <p>We believe designers are thinkers as much as they are makers. So we created the design newsletter we have always wanted to receive.</p> </td>
```
Comment (jfmartinz) [inline UI-UX Design/README.md:296]: Please remove the `p` inside of the `td`. It is not needed.

File: UI-UX Design/README.md [hunk 2]
```diff
@@ -289,8 +289,60 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
...
+      </tr>
+      <!-- second one -->
+      <tr> 
+      <td><a href="https://theme-selection.medium.com/the-10-best-newsletters-for-ui-ux-designers-in-2021-45bbdc4dec01">The Smashing email Newsletter
+</a> </td>
+      <td>Useful front-end & UX bits, delivered once a week.
+      <p>With tools to help you get your work done better. Subscribe and get Vitaly’s Smart Interface Design Checklists PDF via email.</p> </td>
```
Comment (jfmartinz) [inline UI-UX Design/README.md:303]: This one also

File: UI-UX Design/README.md [hunk 3]
```diff
@@ -289,8 +289,60 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
...
+      <td><a href="https://sidebar.io/">Sidebar</a> </td>
+      <td>
+      What can web designers and developers do to build a more sustainable web?
+      <p>Say no to Sidebar-less inboxes Make your inbox happy with Sidebar's daily newsletter.</p> </td>
+      </tr>
+      <!-- fourth one -->
+      <tr>
+      <td> <a href="https://web-design-weekly.com/">Web Design Weekly </a> </td>
+      <td> Stay up-to-date and elevate your web design skills
+      <p>Articles and tutorials on web design and development, delivered every Wednesday, for free. </p></td>
```
Comment (jfmartinz) [inline UI-UX Design/README.md:316]: Another

File: UI-UX Design/README.md [hunk 4]
```diff
@@ -289,8 +289,60 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
...
+</a> </td>
+      <td>Useful front-end & UX bits, delivered once a week.
+      <p>With tools to help you get your work done better. Subscribe and get Vitaly’s Smart Interface Design Checklists PDF via email.</p> </td>
+      </tr>
+      <!-- third one -->
+      <tr>
+      <td><a href="https://sidebar.io/">Sidebar</a> </td>
+      <td>
+      What can web designers and developers do to build a more sustainable web?
+      <p>Say no to Sidebar-less inboxes Make your inbox happy with Sidebar's daily newsletter.</p> </td>
```
Comment (jfmartinz) [inline UI-UX Design/README.md:310]: Another

File: UI-UX Design/README.md [hunk 5]
```diff
@@ -289,8 +289,60 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
...
+      <tr>
+      <td> <a href="https://web-design-weekly.com/">Web Design Weekly </a> </td>
+      <td> Stay up-to-date and elevate your web design skills
+      <p>Articles and tutorials on web design and development, delivered every Wednesday, for free. </p></td>
+      </tr>
+      <!-- fifth one -->
+      <tr>
+      <td> <a href="https://designmodo.com/subscribe/">Designmodo</td>
+      <td>Get the latest and most important news about web design and development.
+    <p>Each month our 143,295 readers receive a newsletter packed with the best articles, news and products. Plus, the best job openings available that month from the best companies.</p> </td>
```
Comment (jfmartinz) [inline UI-UX Design/README.md:322]: Another

File: UI-UX Design/README.md [hunk 6]
```diff
@@ -289,8 +289,60 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
...
+      </tr>
+      <!-- sixth one -->
+      <tr> 
+      <td> <a href="https://heydesigner.com/newsletter/"> Hey designer</td>
+      <td> HeyDesigner is the go-to newsletter for product people, UXers, PMs, and design engineers.
+      <p>Design and frontend links delivered to your inbox every day or week. No spam, unsubscribe at anytime. </p> </td>
```
Comment (jfmartinz) [inline UI-UX Design/README.md:328]: Another one

File: UI-UX Design/README.md [hunk 7]
```diff
@@ -289,8 +289,60 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
...
+      </tr>
+<!-- siventh one -->
+<tr>
+<td> <a href="https://the-webdesigner.co/"> The Web Designer</td>
+<td>Become a Better Web Designer
+<p>Weekly insights, tips & strategies for growing your skills and earning more money doing what you love </p> </td>
```
Comment (jfmartinz) [inline UI-UX Design/README.md:334]: Another

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Please change this and then I will merged once your're done. Thank you!

**Comment by jfmartinz** [comment]
Just edit the code and push it here. THanks

**Review by jfmartinz — APPROVED** [review]
Looks good @Chandan-Kumar-Mahato! Thank you! 💯

**Pull request merged**